pepper-apis/dsm-core/src/test/java/org/broadinstitute/dsm/kits/KitTestUtil.java

```diff
@@ -0,0 +1,517 @@
+import org.broadinstitute.dsm.service.admin.UserAdminTestUtil;
+import org.broadinstitute.dsm.util.EasyPostUtil;
+import org.broadinstitute.dsm.util.KitUtil;
+import org.broadinstitute.dsm.util.NotificationUtil;
+import org.broadinstitute.lddp.db.SimpleResult;
+import org.mockito.Mock;
+
+@Slf4j
+@Getter
+public class KitTestUtil {
+
+    protected static final String SELECT_INSTANCE_ROLE = "SELECT instance_role_id FROM instance_role WHERE name = ?;";
+    protected static final String INSERT_DDP_INSTANCE_ROLE = "INSERT INTO ddp_instance_role (ddp_instance_id, instance_role_id) "
+            + " VALUES (?, ?) ON DUPLICATE KEY UPDATE instance_role_id = ?;";
+    protected static final String SELECT_KIT_TYPE_ID = "SELECT kit_type_id FROM kit_type WHERE kit_type_name = ?";
+    protected static final String INSERT_KIT_DIMENSION = "INSERT INTO kit_dimension (kit_width, kit_height, kit_length, kit_weight) "
+            + " VALUES ('6.9', '1.3', '5.2', '3.2') ON DUPLICATE KEY UPDATE kit_width = '6.9';";
+    protected static final String INSERT_KIT_RETURN = "INSERT INTO kit_return_information (return_address_name, return_address_phone) "
+            + " VALUES ('Broad Institute', '1111112222') ON DUPLICATE KEY UPDATE return_address_name = 'Broad Institute';";
+    protected static final String INSERT_CARRIER = "INSERT INTO carrier_service (carrier, service) "
+            + " VALUES ('FedEx', 'FEDEX_2_DAY') ON DUPLICATE KEY UPDATE carrier = 'FedEx';";
+    protected static final String INSERT_DDP_KIT_REQUEST_SETTINGS =
+            "INSERT INTO ddp_kit_request_settings (ddp_instance_id, kit_type_id, kit_return_id, carrier_service_to_id, kit_dimension_id) "
+                    + " VALUES (?, ?, ?, ?, ?) ;";
+    private static final String SELECT_DSM_KIT_REQUEST_ID =
+            "SELECT dsm_kit_request_id from ddp_kit_request where ddp_kit_request_id like ? ";
+
+    private static final String INSERT_KIT_TYPE_ID = "INSERT into kit_type (kit_type_name , display_name) VALUES (?, ?)";
+    private static final String SELECT_BY_DISPLAY_NAME = " and display_name = ? ";
+
+    private static final String INSERT_SUB_KITS_SETTINGS =
+            "INSERT INTO  sub_kits_settings (ddp_kit_request_settings_id, kit_type_id, kit_count, hide_on_sample_pages, external_name) "
+                    + " VALUES (?, ?, 1, ?, '') ;";
+
+    private static final String INSERT_EVENT_TYPE =
+            "INSERT INTO event_type (ddp_instance_id, event_name, event_description, kit_type_id, event_type) "
+                    + " VALUES (?, ?, ?, ?, ?) ;";
+
+    protected UserAdminTestUtil adminUtil = new UserAdminTestUtil();
+    protected Integer ddpInstanceId;
+    protected Integer ddpGroupId;
+    protected Integer ddpInstanceGroupId;
+    protected Integer kitTypeId;
+    protected Integer kitDimensionId;
+    protected Integer kitReturnId;
+    protected  Integer carrierId;
+    protected Integer ddpKitRequestSettingsId;
```

Comment:
These don't need to be protected now that there's not a base class.  I think they can all be private.